### DISCUSSION TITLE
Radial Progress with complementry bar

### DISCUSSION THREAD

**Discussion by pooyaEst** [saadeghi/daisyui#1998] [Ideas / feature requests]
First of all thx for this this awesome library. 

i think if there be an optional complementary color for Radial Progress, that would be very nice and expand it's use cases. 

for example in case of a voting system, green can refer to those in favor and red to those who voted no.

**Comment by NikarashiHatsu** [comment]
I think that's not how radial progress meant to be used, I recommend you to use chart library such Chart.js or ApexChart to achieve such feature.

**Reply by pooyaEst** [reply]
even as a progress bar having two colors to indicate how much job has been done and how much is remaining would look neat

**Comment by saadeghi** [comment]
> I think that's not how radial progress meant to be used, I recommend you to use chart library such Chart.js or ApexChart to achieve such feature.

Agree.  
It's just a one-value progress. More than one value would be donut chart.
